PendingReleaseNotes

```diff
@@ -110,6 +110,12 @@ CephFS: Disallow delegating preallocated inode ranges to clients. Config
   and valid), diff-iterate is now guaranteed to execute locally if exclusive
   lock is available.  This brings a dramatic performance improvement for QEMU
   live disk synchronization and backup use cases.
+* RGW: topic policies needs to be set for anyone other than the topic owner,
+  to modify, delete or get the topic. with the exception of: 
+  * topics that were created in an older version that did not store the owner as 
+  part of the topic's configuration
+  * anyone may be allowed to publish to a topic, even without proper policy,
+  based on the value of the ``mandatory_topic_permissions`` configuration parameter.
```

Comment:
Notification topics are now owned by the user that created them. By default, only the owner can read/write their topics. Topic policy documents are now supported to grant these permissions to other users.

Preexisting topics are treated as if they have no owner, and any user can read/write them using the SNS API. If such a topic is recreated with CreateTopic, the issuing user becomes the new owner.

For backward compatibility, all users still have permission to publish bucket notifications to topics owned by other users. A new configuration parameter ``mandatory_topic_permissions`` can be enabled to deny `sns:Publish` permissions unless explicitly granted by topic policy.

(p.s. let's rename `mandatory_topic_permissions` to something like `rgw_topic_require_publish_policy` before the release?)